### ISSUE TITLE
1.9.1 Forever loading history or contacts view

### ISSUE THREAD

**Issue by danisOktaxi** [ringcentral/ringcentral-embeddable#709]
I update to use the latest version 1.9.1 and widget keeps loading forever on history or contacts view.

![image](https://github.com/ringcentral/ringcentral-embeddable/assets/83176592/51e6a64d-a8d7-4269-8de3-c353815b2c30)

I see this error in console
```
Uncaught (in promise) ReferenceError: pageNum is not defined
    at mt._fetchAll (index.ts:28:29)
    at async mt._sync (index.ts:47:7)
_fetchAll @ index.ts:28
```

**Comment by embbnux** [comment]
hi @danisOktaxi thanks for the feedback. Got the root cause, will fix it today

**Comment by embbnux** [comment]
@danisOktaxi fixed at 1.9.2

**Comment by danisOktaxi** [comment]
Thank you. Tested and working now.
